utils/get_modified_files.py

```diff
@@ -24,7 +24,7 @@
 import sys
 
 
-fork_point_sha = subprocess.check_output("git merge-base main HEAD".split()).decode("utf-8")
+fork_point_sha = subprocess.check_output("git merge-base master HEAD".split()).decode("utf-8")
```

Comment:
This file should not be changed (as we renamed master to main).

Reply:
Done